From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Subject: avoid unstable-curve448

See the discussion in rust-pgp's debian/patches about avoiding
unstable-curve448.  When that is resolved, we can drop this.

Forwarded: not-needed

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -52,9 +52,3 @@
 
 [dependencies.sop]
 version = "0.8"
-
-[features]
-unstable-curve448 = [
-    "pgp/unstable-curve448",
-    "rpgpie/unstable-curve448",
-]
